example-serverless-app-reuse/reuse-auth-only.yaml

```diff
@@ -64,10 +116,19 @@ Resources:
             Id: dummy-origin
             CustomOriginConfig:
               OriginProtocolPolicy: match-viewer
-          - DomainName: change.this.to.your.origin
+          - DomainName: !Sub "${ApplicationBucket}.s3.amazonaws.com"
             Id: protected-origin
-            CustomOriginConfig:
-              OriginProtocolPolicy: match-viewer
+            S3OriginConfig:
+              OriginAccessIdentity: !Sub "origin-access-identity/cloudfront/${CloudFrontOriginAccessIdentity}"
+        CustomErrorResponses:
+            - ErrorCode: 404
+              ResponseCode: 200
+              ResponsePagePath: /index.html
+            - ErrorCode: 403
+              ResponseCode: 200
+              ResponsePagePath: /index.html
+        PriceClass: PriceClass_100
```

Comment:
Think it is better to remove this too - cater for users in other regions of the World

Reply:
Good point.  It really was temp code to save testing time as the refresh for world takes *forever*

Reply:
Interesting solution to use the report only CSP